internal/graph/lookupresources2.go

```diff
@@ -145,9 +146,7 @@ func (crr *CursoredLookupResources2) afterSameType(
 			spiceerrors.DebugAssert(func() bool {
 				return err == nil
 			}, "Error in entrypoint.DebugString()")
-			ctx, span := tracer.Start(ctx, "entrypoint", trace.WithAttributes(
-				attribute.String("entrypoint", ds),
-			))
+			ctx, span := tracer.Start(ctx, "entrypoint: "+ds, trace.WithAttributes())
```

Comment:
Is this to help with filtering?

Reply:
Basically to make the traces easier to navigate.